doc/Caliptra.md

```diff
@@ -87,6 +87,14 @@ In this version of the specification, the desired capabilities address the basic
 
 Caliptra implements the DICE Protection Environment (DPE) API, allowing it to derive and wield a DICE identity on behalf of other elements within the SoC. Use cases for this API include serving as a signing oracle for a Security Protocol and Data Model (SPDM) responder that is executing in the SoC application processor, as well as authentication to a discrete TPM device.
 
+### SVN-bound key derivation via hash chains
```

Comment:
could you add this service to the "Service surface" section around line 631?

Reply:
Done